### PR TITLE
Make portfolio using datastore to store and load comments

### PR DESCRIPTION
**Pull request by yashasvi25**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,20 +30,26 @@
...
 
     String text = getParameter(request, "comment-input", "");
-    cmtarr.add(text);
+    long timestamp = System.currentTimeMillis();
+
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("comment-input", text);
+    taskEntity.setProperty("timestamp", timestamp);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```
Comment (rahugur) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:51]: Will creating this object only once makes sense then put this into init() and create object only once rather on new on every request.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -27,20 +30,26 @@
...
 public class DataServlet extends HttpServlet { 
 
-  ArrayList<String> cmtarr = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
-    String json = new Gson().toJson(cmtarr);
-    response.getWriter().println(json);
+    //String json = new Gson().toJson(cmtarr);
```
Comment (rahugur) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:37]: doGet is not complete right?

**Comment by rahugur** [comment]
Give PR description meaningful name. Currently it's "Making portfolio" and prev also had same name.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (yashasvi25) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:37]: Yes, these commits were only for storing comments in the datastore

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -36,23 +21,23 @@ $(document).ready(function () {
 function getMessages() {
   fetch('/data').then(response => response.json()).then((cmts) => {
 
-
+    console.log(cmts);
```
Comment (rahugur) [inline portfolio/src/main/webapp/script.js:24]: this can be removed.

**Pull request merged**